USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -112,20 +194,76 @@ private string RenderMarker(Marker input)
                     output.AppendLine("<div class=\"header\">");
                     output.Append(hMarker.HeaderText);
                     output.AppendLine("</div>");
+                    
                     break;
                 case MTMarker mTMarker:
                     output.AppendLine("<div class=\"majortitle\">");
                     output.AppendLine(mTMarker.Title);
                     output.AppendLine("</div>");
+                    foreach (Marker marker in input.Contents)
+                    {
+                        output.Append(RenderMarker(marker));
+                    }
+
+                    if (!separateChapters)   // No double page breaks before books
+                    {
+                        output.AppendLine("<br class=\"pagebreak\"></br>");
+                    }
+                    break;
+                case FMarker fMarker:
+                    StringBuilder footnote = new StringBuilder();
+                    fMarker.FootNoteID = $"{(char)(FootnoteTextTags.Count+97)}";

```

Comment:
Clever.

Reply:
## Just a note
2 Peter 2:13  eng_ulb
```
\f + \ft Instead of \fga They enjoy their deceitful actions while they are feasting with you. \fqa* , other modern translations have something like, \fqa They enjoy their actions while they are feasting with you in love feasts  \fqa* . \f*
```
\fga isn't a marker ... messed up my rendering.